arm-compute/compositeComputeClient.json

```diff
@@ -4,7 +4,7 @@
         "description": "Composite Swagger for Compute Client"
     },
   "documents": [
-    "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-compute/2015-06-15/swagger/compute.json",
+    "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-compute/2016-03-30/swagger/compute.json",
```

Comment:
It's a deduction that AR shall be able to figure out -- the location of other relative specs shall be based on the composite file.